include/ofi_hmem.h

```diff
@@ -87,8 +87,7 @@ ssize_t ofi_copy_to_hmem_iov(const struct iovec *hmem_iov,
 			     size_t hmem_iov_count, uint64_t hmem_iov_offset,
 			     void *src, size_t size);
 
-int ofi_hmem_init(enum fi_hmem_iface iface);
-
+void ofi_hmem_init(void);
```

Comment:
Hmm... I expected to see callers updated for this change, but then I don't see any existing callers.

Reply:
Yeah. In future verbs PR, I was calling this. I was having verbs call ofi_hmem_init() when verbs saw an MR of a specific HMEM iface type. Since running without FI_MR_HMEM requires all HMEM ifaces to be initialized ahead of them, I think this approach would also work with FI_MR_HMEM.